saleor/graphql/meta/mutations/utils.py

```diff
@@ -15,12 +16,22 @@ def get_valid_metadata_instance(instance) -> ModelWithMetadata:
     return instance
 
 
+TYPE_UPDATED_FIELD = {
+    "Checkout": "last_change",
+    "Order": "updated_at",
+    "Product": "updated_at",
+    "ProductVariant": "updated_at",
+    "TransactionItem": "modified_at",
+    "User": "updated_at",
+    "Promotion": "updated_at",
+    "Payment": "modified_at",
+}
```

Comment:
Looks good to me. We can think about approaching `TYPE_UPDATED_FIELD` differently, as it is possible we will forget to update it when a new model is introduced. Maybe by moving information about the "updated_at" field name to a custom `Model.Meta` field? We could then raise an exception in `get_extra_update_field` if the field name is not set, which will allow us to automatically catch it in tests.